src/app/complex/complex-details/complex-pairwise-relationships/complex-pairwise-relationship/complex-pairwise-relationship.component.html

```diff
@@ -8,12 +8,19 @@ <h5 class="float-left">
         {{covariationCoveragePercentage()}}% of the protein-protein pairs in the complex have {{dataType()}} data in {{databaseName()}}.
       }
     </h5>
+    <div class="columns medium-12 no-pad-left no-pad-right">
+        Hover over the heatmap cells to see the {{dataType()}} scores for each protein pair.
+    </div>
+    <div class="columns medium-12 no-pad-left no-pad-right">
+      Click on the heatmap cells to open a link to {{databaseName()}} with the data for the protein pair.
+    </div>
```

Comment:
This can be reworded if anybody has any suggestion. Also, we can change the style.